src/interface/read_config_file.py

```diff
@@ -0,0 +1,108 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, fileconfig): 
+            
+            
+            self.xmld = etree.parse(fileconfig)             
+            self.root = self.xmld.getroot()
+
+        
+
+    def get_output_file(self):
+        """ 
+        Returns the outputfile path, name and extension values from config file
+        listoutput: variable to assign all values read from config file
+        """
+        list_output = []
+        for output in self.root.findall('Output'):
+            path = output.find('Path').text
+            name = output.find('Name').text         
+            extension = output.find('Extension').text   
+
+
+        list_output.append(path)
+        list_output.append(name)
+        list_output.append(extension)
+
+        
+        return list_output
+
+    def list_of_file_types_supported(self):
+        """     
+        Returns a list with all files types values supported, it will be also get from config file
+        listtypes: variable of list type, it will have assigned all supported files, this info is read from config file
+        """
+        list_types = []
+        
+        for solve in self.root.findall('./Output/Extension/type'):          
+            name_alg = solve.find('type').text
+            #print namealg
+            #listypes.append(typevalue)
+        
+
+    def get_list_of_algorithms(self):
+        """
+        Returns a list with all availables algorithms from config file, and the first item will always return
+        the active algorithm set for the game.
+        listalgorithms: variable of list type, it will have assigned all algorithms read from config file
```

Comment:
the parameter described here is not the same used in the code, consider use an empty line between the parameters and your description it will help to the readability